MMM-COVID19.js

```diff
@@ -207,23 +240,31 @@ Module.register("MMM-COVID19", {
         if (newDeaths) {
           newDeathsCell.innerHTML = '+' + newDeaths
         }
+        seriousCell.className = 'number deaths ' + this.config.infoRowClass
```

Comment:
@testdruif wrong className used here. Don't use the same color as Deaths. If you want to highlight it someway, please add a new class called `serious` and add the proper entry at the .css file of the module